<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>208</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>208</width>
    <height>485</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>来聊</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icons/icon/Chat-icon.png</normaloff>:/icons/icon/Chat-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>447</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icons/icon/search-icon-hi.png</normaloff>:/icons/icon/search-icon-hi.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>在线</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/QQ/icon/qq/Header+Search_Online_Normal@2x.png</normaloff>:/QQ/icon/qq/Header+Search_Online_Normal@2x.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>离线</string>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/QQ/icon/qq/Header+Search_Offline_Hover@2x.png</normaloff>:/QQ/icon/qq/Header+Search_Offline_Hover@2x.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_tx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>80</y>
      <width>210</width>
      <height>360</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>360</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>360</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <column>
     <property name="text">
      <string>我的好友</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>210</width>
      <height>485</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/icons/icon/chat.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>searchBtn</zorder>
   <zorder>comboBox</zorder>
   <zorder>nameLabel</zorder>
   <zorder>toolButton_tx</zorder>
   <zorder>treeWidget</zorder>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除好友</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>编辑备注</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>Client</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
